server/src/main/java/org/jboss/as/server/deployment/module/ResourceRoot.java

```diff
@@ -104,4 +105,17 @@ public void merge(final ResourceRoot additionalResourceRoot) {
             this.exportFilters.addAll(additionalResourceRoot.getExportFilters());
         }
     }
+
+    /**
+     * Checks if the additional Resource root exists
+     *
+     * @param additionalResourceRoot check is performed on this fix for WFCORE-4231
+     */
+
+    public boolean resourceRootExistenceCheck() throws StartException {
+    if(this.getRoot()==null || !this.getRoot().exists()) {
+    throw ServerLogger.ROOT_LOGGER.additionalResourceRootDoesNotExist(this.getRoot().getPathName());
```

Comment:
If the general changes are approved the formatting will need revisiting here, this should be indented using 4 spaces.

The ide-configs folder contains formatting templates for most IDEs